book/website/collaborating_github/reviewing-contributions.md

```diff
@@ -0,0 +1,89 @@
+These ideas and discussions lead to better quality code and opens new perspectives regarding a particular feature. The reviews are done to make high quality production level code.  
+Maintenance deals with keeping the existing code updated. It requires that whenever a new feature is added or whenever a new idea is accepted, the corresponding changes must be made in existing codebase(if required) and the documentation.  
+It requires [opening](https://help.github.com/en/github/managing-your-work-on-github/creating-an-issue),[deleting](https://help.github.com/en/github/managing-your-work-on-github/deleting-an-issue) and [closing/linking](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) important issues, [reviewing](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request), [labelling](https://help.github.com/en/github/managing-your-work-on-github/labeling-issues-and-pull-requests), [changing the stages](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-stage-of-a-pull-request), [merging](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/merging-a-pull-request) and [closing](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/closing-a-pull-request) important pull requests.  
+Knowing the working of Version control systems, [markdown](https://guides.github.com/features/mastering-markdown/) and open source [workflow](https://guides.github.com/introduction/flow/) helps to a great extent in carrying out all these tasks.
+
+## Ownership and Permissions 
+In any team or organization owned project, there are different contributors. 
+
+- Some start the project and are generally, the owners of the project. The Codeowners who have the rights over who can make reviews and suggest changes to contributions. The individuals or teams who are codeowners for a certain project are specified at the time of project creation in a [file](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) of the project.  
+They can make other contributors as maintainers, reviewers or members working on the project. They can also involve other codeowners and are responsible for granting permissions. 
+
+- Maintainers are involved with creating issues and pull requests whenever required. They keep the codebase updated and help in reviewing contributions. They can merge pull requests.  
+They can also request reviews from someone else. There are various [levels of permissions](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization) for a project in GitHub.
+
+## Reviewing process
```

Comment:
This section is quite focussed on testing this work and committing changes. I'd like to also acknowledge that PRs are conversations/exchanges of knowledge where discussions of implementation and future goals may also happen, more than "does this run?", if that's ok?

Reply:
I kept this section just to introduce that we have different access rights and permission levels in a collaborative open source project.This section was created keeping just the basics in mind. I have explained all about the communication and discussions part in the Guidelines. I can write a line or two about that here too, if you insist.

Reply:
I think saying something like "This section covers access rights and permission levels. For more information on communications and discussions, please see [LINK TO GUIDELINES]" is all that's needed here :)